client/ayon_core/hosts/max/api/lib.py

```diff
@@ -356,6 +357,26 @@ def get_max_version():
     return max_info[7]
 
 
+def get_current_task_entity():
+    """Function to get current task entity data
+
+    Returns:
+        dict: data of task entity
+    """
+    project_name = get_current_project_name()
+    folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
+    folder_entity = ayon_api.get_folder_by_path(
+        project_name, folder_path, fields={"id"})
+    task_entity = ayon_api.get_task_by_name(
+        project_name,
+        folder_entity["id"],
+        task_name,
+        fields={"attrib"}
+    )
+    return task_entity
```

Comment:
I understand the re-use need of this - but I feel it's a bit odd that this is inside the max lib specifically. If we really need to do this often I feel like this is better implemented more globally in ayon core or in the ayon api. @iLLiCiTiT thoughts?

Also, the fact that this library function defaults to filtering fields to `attrib` only feels a bit confusing because the function name seems to hint it'll just return the task entity.